app/models/article.server.model.js

```diff
@@ -25,7 +21,19 @@ var ArticleSchema = new Schema({
 		default: '',
 		trim: true
 	},
-	user: {
+	updatedAt: {
+		type: Date,
+		default: Date.now
+	},
+	updatedBy: {
+		type: Schema.ObjectId,
+		ref: 'User'
+	},
+	createdAt: {
```

Comment:
I just went for uniformity, as the updated/created could also be used for booleans. The At suffix implies time:

article.updated: true
or 
if (article.updated) {}

vs.

article.updatedAt = Date.now();
or
if(article.updatedAt > someOtherDate) {}

If you want I can change them, but I plan to update the generators so that every new model has those fields, and in that case by using the At and By suffixes, we also avoid name collisions if the developer want to use created or updated fields for other things (as booleans for example).

What do you think?